base_custom_info/models/custom_info_category.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class CustomInfoCategory(models.Model):
+    _description = "Categorize custom info properties"
+    _name = "custom.info.category"
+    _order = "sequence, name"
+
+    name = fields.Char(index=True, translate=True, required=True)
+    sequence = fields.Integer(index=True)
+    property_ids = fields.One2many(
+        comodel_name="custom.info.property",
+        inverse_name="category_id",
+        string="Properties",
+        help="Properties in this category.",
+    )
+
+    @api.multi
+    def check_access_rule(self, operation):
+        """You access a category if you access at least one property."""
+        last = None
+        for prop in self.mapped("property_ids"):
+            try:
+                prop.check_access_rule(operation)
+                break
+            except Exception as last:
+                pass
+        if last:
+            raise last
```

Comment:
The docstring says that you need to access at least one property. Imagine you can access the 2nd one in the recordset. Doing what you say would forbid access when the 1st one fails. However there's one failure, we should `return` instead of `break`ing.